db/blip_handler.go

```diff
@@ -355,6 +357,10 @@ func (bh *blipHandler) sendBatchOfChanges(sender *blip.Sender, changeArray [][]i
 		go func(bh *blipHandler, sender *blip.Sender, response *blip.Message, changeArray [][]interface{}, sendTime time.Time, database *Database) {
 			if err := bh.handleChangesResponse(sender, response, changeArray, sendTime, database); err != nil {
 				base.ErrorfCtx(bh.loggingCtx, "Error from bh.handleChangesResponse: %v", err)
```

Comment:
Previously this was only being invoked for SG-side errors, so ErrorfCtx made sense.  Are there client-side errors that will trigger this error logging now?